types/oidc-provider/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for oidc-provider 7.8
 // Project: https://github.com/panva/node-oidc-provider
 // Definitions by: Filip Skokan <https://github.com/panva>
+//                 Matt R. Wilson <https://github.com/mastermatt>
```

Comment:
Please don't add yourself as maintainer.

Reply:
'Definitions by' doesn't denote maintainers, just who has contributed over time.

Reply:
It denotes who can approve and trigger merges via comments on DefinitelyTyped.

Reply:
I've removed this, but as the maintainer of the lib, I'd recommend bundling your types if you want that level of control.
DT tends to be community driven.

Reply:
Imposing @types transient dependencies such as @types/koa on users who don't use TS is not acceptable, which is why i moved my definitions here, to be opt-in.